src/main/java/core/basesyntax/List.java

```diff
@@ -14,7 +16,7 @@
 
     T remove(int index);
 
-    T remove(T t);
+    T remove(T t) throws FileNotFoundException;
```

Comment:
should this method have this exception in signature?

Reply:
I don't even know how it appeared there)